framework/src/modules/chain/transport/transport.js

```diff
@@ -235,27 +234,10 @@ class Transport {
 		}
 
 		try {
-			const data = await this.blocksModule.loadBlocksDataWS({
-				limit: 34, // 1977100 bytes
-				lastId: query.lastBlockId,
-			});
-
-			_.each(data, block => {
-				if (block.tf_data) {
-					try {
-						block.tf_data = block.tf_data.toString('utf8');
-					} catch (e) {
-						this.logger.error(
-							'Transport->blocks: Failed to convert data field to UTF-8',
-							{
-								block,
-								error: e,
-							},
-						);
-					}
-				}
-			});
-
+			const data = await this.blocksModule.loadBlocksFromLastBlockId(
```

Comment:
Param `storage` missing. Maybe use object as param?

Reply:
this function doesn't require storage as input (it's managed in blocks module)